db/migrate/20220122052009_user_belt_to_integer.rb

```diff
@@ -0,0 +1,7 @@
+class UserBeltToInteger < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :users, :belt
+    add_column :users, :belt, :integer
```

Comment:
Instead of removing and adding the column, you can use `change_column` instead

Reply:
Also. Instead of creating a new migration for this, why don't you just update the one were you create [belt](https://github.com/bryanmg/ammdk_colima/pull/6/files#diff-a25e064d3d4458bbbdc2598b6e1b3064515d9bc6eeb5883abe5a9f2643790bc4R5) column?